packages/components-angular/.eslintrc.json

```diff
@@ -0,0 +1,40 @@
+{
+  "root": true,
+  "ignorePatterns": ["projects/**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "parserOptions": {
+        "project": ["tsconfig.json"],
+        "createDefaultProgram": true
```

Comment:
Not sure if this will work for IntelliJ and vscode where we also had to specify the base directory dynamically with `__dirname`. Did you check @imagoiq?

Reply:
It looks ok at first sight at least. I didn't see any errors.